src/components/Dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import Text from '../Text/Text';
+import { Link } from 'react-router-dom';
+import Grade from '../Grade/Grade';
+import styles from './dashboard.module.scss';
+import BarChart from 'components/Dashboard/BarChart'
+import PieChart from 'components/Dashboard/PieChart'
+import LineChart from 'components/Dashboard/LineChart'
+import DoubleBarChart from 'components/Dashboard/DoubleBarChart'
+import add from 'assets/img/plus.svg';
+const SCHOOLS = [
+    { name: 'Ballard High School', rank: '27' },
+    { name: 'Center High School', rank: '1' },
+    { name: 'Cleveland High School', rank: '92' },
+    { name: 'Center School', rank: '74' },
+
+]
+const Dashboard = ({ dashboardTitle, reportCard, data }) => {
```

Comment:
Overall looks good, but this markup should be divided into separate components

We can merge it as is for now and have a separate PR to divide it into semantic components without having so much markup all in one component. Happy to discuss further how we could go about doing this if you want